tests/unit/acquisition/test_function.py

```diff
@@ -85,14 +90,37 @@ def test_expected_improvement_builder_builds_expected_improvement(
     npt.assert_array_almost_equal(acq_fn(query_at), expected)
 
 
-def test_expected_improvement() -> None:
-    def _ei(x: tf.Tensor) -> tf.Tensor:
-        n = tfp.distributions.Normal(0, 1)
-        return - x * n.cdf(-x) + n.prob(-x)
+@random_seed()
+@pytest.mark.parametrize('best, rtol, atol', [
+    (tf.constant([50.0]), 0.01, 1e-3),
+    (BRANIN_GLOBAL_MINIMUM[None], 0.01, 1e-3),
+    (BRANIN_GLOBAL_MINIMUM[None] * 1.01, 0.01, 1e-3)
+])
+@pytest.mark.parametrize('variance_scale', tf.constant([0.1, 1.0, 10.0], dtype=tf.float64))
+def test_expected_improvement(
+    variance_scale: tf.Tensor, best: tf.Tensor, rtol: float, atol: float
+) -> None:
+    best = tf.cast(best, dtype=tf.float64)
+
+    x_range = tf.linspace(0.0, 1.0, 11)
+    x_range = tf.cast(x_range, dtype=tf.float64)
+    xs = tf.reshape(tf.stack(tf.meshgrid(x_range, x_range, indexing='ij'), axis=-1), (-1, 2))
```

Comment:
make a separate function to do this?

Reply:
you have a create grid function in plotting utilities that does just this

Reply:
yeah. I was thinking of porting that over. An alternative is to sample from a `Box`.

Reply:
but I'd have to write tests for the former

Reply:
created issue #42